Tests/UnitTests/UT_Wallet.cs

```diff
@@ -0,0 +1,122 @@
+using DOL.GS.Finance;
```

Comment:
I think it can be a good idea to add a concurrent test (like 10 threads adding/removing money), I don't think we can do a perfect test for that but if we can catch some issues, it's always better :)

PS: I put this comment here, maybe it's better in integration tests.

Reply:
Testing concurrency is a bit tricky. It could be an `Explicit` test at best, but you wouldn't want to add it for everything either, so it's mostly for personal use (which is fine).